File: 프로그래머스/2/12939. 최댓값과 최솟값/최댓값과 최솟값.py
def solution(s):
    answer = ''
    l = list(map(str, s.split()))
    
    for i in range(len(l)) :
        l[i] = int(l[i])
    
    answer += str(min(l))
    answer += ' '
    answer += str(max(l))
    
    return answer

File: 프로그래머스/2/12939. 최댓값과 최솟값/최댓값과 최솟값.cpp
#include <string>
#include <vector>
#include <algorithm>
#include <sstream>

using namespace std;

string solution(string s) {
    string answer = "";
    vector<int> v;
    stringstream ss(s);
    int num;
    while (ss>>num)
      v.push_back(num);
    
    int maxvalue = *max_element(v.begin(), v.end());
    int minvalue = *min_element(v.begin(), v.end());
        
    answer = to_string(minvalue) + " " + to_string(maxvalue);
    
    return answer;
}